wolfcrypt/src/port/nxp/ksdk_port.c

```diff
@@ -974,7 +974,7 @@ static const ECPoint ecBasePoint = {
     0x1e, 0xe0, 0xb4, 0x86, 0xa0, 0xb8, 0xa1, 0x19, 0xae, 0x20},
 };
 
-const ECPoint *wc_curve25519_GetBasePoint(void)
+const ECPoint *nxp_ltc_curve25519_GetBasePoint(void)
```

Comment:
In this file getting the following error:

```
6> /Users/davidgarske/GitHub/wolfssl/wolfcrypt/src/port/nxp/ksdk_port.c:1148:9: error: 'curve25519_bCurveParam' undeclared (first use in this function); did you mean 'curve25519_aCurveParam'?
```

Traced it down to typo. Should be `curve_bCurveParam`. However let's rename it to `curve25519_bCurveParam` for consistency with the `a` param at line 988.

Reply:
Thanks Dave for catching all the nuances and warts in the NXP build -- I'll push those changes shortly.  Just getting rolling here -- you start early!

Reply:
That typo snuck in at aadec345ab894e9ca5304885964f36c64963661e "45 files changed, 1709 insertions(+), 610 deletions(-)".  I fear there's other easter eggs in that PR waiting to hatch...